### ISSUE TITLE
Add docs section for uses of Yolov5 in the wild

### ISSUE THREAD

**Issue by robmarkcole** [ultralytics/yolov5#3309]
## 🚀 Feature
Add docs section for uses of Yolov5 in the wild, to enable new users to quickly understand the use cases for yolo and link up with people already putting it into production

## Motivation
Accelerate adoption of yolov5 

## Pitch
For example I discovered [TowerScout](https://groups.ischool.berkeley.edu/TowerScout/index.html) and am keen to discover other uses cases in the aerial imaging domain

## Alternatives
NA

## Additional context
NA

**Comment by glenn-jocher** [comment]
@robmarkcole hey thanks for the docs idea! We actually just got started on some docs at https://docs.ultralytics.com/

Hopefully in the next few weeks we'll start generating more content and updating the docs!

I looked at TowerScout, very cool! They are using YOLOv5 coupled with an EfficientNet in a two stage system for detection (and reduction of False Positives) of cooling towers.
https://groups.ischool.berkeley.edu/TowerScout/index.html
